bug-bounty-list/bug-bounty-list.json

```diff
@@ -8262,17 +8172,17 @@
   {
     "program_name": "Zomato",
     "policy_url": "https://www.zomato.com/security",
-    "submission_url": "",
+    "submission_url": "security@upserve.com",
     "launch_date": "",
     "cash_reward": true,
     "swag": false,
     "hall_of_fame": true,
     "safe_harbor": "full"
   },
   {
-    "program_name": "Zscaler",
+    "program_name": "zscaler",
```

Comment:
I noticed a number of conversions to lowercase -- was that intentional? If so, you should ensure the others are also lowercase.